docs/dictionary/command/visual-effect.lcdoc

```diff
@@ -15,21 +15,21 @@ OS: mac, windows, linux, ios, android, html5
 Platforms: desktop, mobile
 
 Example:
-visual effect dissolve
+visual effect "dissolve"
 go next card
 
 Example:
 -- This handler in an object will cause to hide then show 
 --  with a visual effect when clicked.
 on mouseUp
   hide me
-  show me with visual effect iris close slow
+  show me with visual effect "iris" close slow
```

Comment:
Hello,
Thank you for pointing this out. I will look into it and update the PR accordingly.